webpack/scenes/ContentViews/Details/Repositories/LastSync.js

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment } from 'react';
+import { CheckCircleIcon, ExclamationTriangleIcon, CloseIcon } from '@patternfly/react-icons';
+import { Link } from 'react-router-dom';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import PropTypes from 'prop-types';
+
+const LastSync = ({ lastSyncWords, lastSync }) => {
+  if (lastSync && lastSyncWords) {
+    let Icon;
+    let color = 'black';
+    const { result, id } = lastSync;
+
+    if (result === 'success') {
+      Icon = CheckCircleIcon;
+      color = 'green';
+    } else if (result === 'warning') {
+      Icon = ExclamationTriangleIcon;
+      color = 'orange';
+    } else if (result === 'error') {
+      Icon = CloseIcon;
+      color = 'red';
+    } else {
+      Icon = Fragment;
+    }
+
+    return (
+      <Link to={urlBuilder('foreman_tasks/tasks', '', id)}>
```

Comment:
> But I think it's simpler just to change it to a good old-fashioned <a>. (Since we're switching between plugins, the Redux store gets cleared out on page load anyway.) @laviro thoughts?

since Katello is using `react-router-dom` it makes sense to use the `Link` from that lib,
although it would be awesome if soon the routes will be registered through Foreman, and than that `push` should be used, there is an helper @amirfefer wrote: [pushUrl](https://github.com/theforeman/foreman/blob/88eb8d4846c2806cde40253e8823e5f1f9a493cd/webpack/assets/javascripts/foreman_navigation.js#L34) which is easier.

Reply:
I switched them to `a`'s for now, I'm not exactly sure why `Link` isn't working, but I'm guessing it's some sort of cross-application communication issue. 

Any reason not to use `a`? The back button seems to work fine, but not sure if there are other issues.

I can look into adding the `#tabname` part of the url too, that would be nice.

Reply:
The advantage of using `Link` is that it preserves the current browser document, and thus the Redux store, etc.  It does this by using `window.location.history`, as opposed to `<a>` which causes the browser to do a new `GET` request to that url.

If we're traveling to a different Foreman plugin I don't see a reason not to use `<a>`.